test/browser.js

```diff
@@ -3,7 +3,7 @@
 
 // require('./exchange-files')
 // require('./pubsub')
-require('./kad-dht')
+// require('./kad-dht')
```

Comment:
This means it is not being tested

Reply:
Yes, more info: https://github.com/ipfs/interop/pull/36#pullrequestreview-188608872

I can have a look after to that issue with `id()`, but I think that it should not be a blocker for this. I can create an issue to track it, as I think it is better to have these tests in now, than have nothing

Reply:
https://github.com/ipfs/interop/pull/36#discussion_r244681154